.buildkite/pipeline-resource-definitions/cloud-security-posture/cspm-agentless-scout.yml

```diff
@@ -0,0 +1,59 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: bk-kibana-cspm-agentless-scout
+  description: 'CSPM Agentless Scout E2E Tests - Cloud Security Posture Management agentless integration tests using Scout and Playwright'
+  links:
+    - url: 'https://buildkite.com/elastic/kibana-cspm-agentless-scout'
+      title: Pipeline link
+spec:
+  type: buildkite-pipeline
+  system: buildkite
+  owner: 'group:kibana-cloud-security-posture'
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / CSPM Agentless Scout
+      description: 'Runs CSPM Agentless E2E tests with Scout and Playwright for both Stateful and Serverless Security deployments'
+    spec:
+      env:
+        SLACK_NOTIFICATIONS_CHANNEL: '#cloud-security-posture-ci'
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'false'
+
+      allow_rebuilds: true
+      branch_configuration: main
+      default_branch: main
+      repository: elastic/kibana
+      pipeline_file: .buildkite/pipelines/cloud_security_posture/cspm_agentless_scout.yml
+      provider_settings:
+        build_branches: true
+        build_pull_requests: true
+        publish_commit_status: true
```

Comment:
To date we haven't used this in the Kibana repo, instead using the build bot.  It's probably fine, but we'll want to keep an eye on it to verify.